homeassistant/components/sensor/min_max.py

```diff
@@ -47,9 +47,11 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_TYPE, default=SENSOR_TYPES[ATTR_MAX_VALUE]):
         vol.All(cv.string, vol.In(SENSOR_TYPES.values())),
-    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
+    vol.Optional(CONF_ID): cv.string,
```

Comment:
This doesn't add real value. The entity IDs are for identifying the entity and should only be set if we are not able to determine it in another way.